packages/flutter_tools/lib/src/ios/fallback_discovery.dart

```diff
@@ -159,14 +159,10 @@ class FallbackDiscovery {
         );
         final VM vm = await vmService.getVM();
         for (final IsolateRef isolateRefs in vm.isolates) {
-          final dynamic isolateResponse = await vmService.getIsolate(
+          final Isolate isolateResponse = await vmService.getIsolate(
             isolateRefs.id,
```

Comment:
what does getIsolate return if this id does not exist, and how should that error case be handled?